### ISSUE TITLE
Please do not close instructional team created issues on repos that you did not open.

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/meta#25]
Issues should be closed by the person who opened them when the change has been made or consensus  is reached.

**Comment by jrhorn424** [comment]
:+1:

**Comment by GA-MEB** [comment]
So just to confirm: with every issue that we leave on every repo, we need to find whoever made the issue and get them to close it? What if you asked someone to make an issue for you?

**Comment by jrhorn424** [comment]
I don't believe that was @gaand's intent. I believe the intent is for everyone to be less reckless when closing issues.
- The outcome of one issue sometimes depends on other issues.
- Issues are not (always) tasks.

That means sometimes we need to decide as a group when issues are closed. There seems to be some missing context here for what triggered this discussion. We should probably take this offline since face-to-face has higher bandwidth.

**Comment by jrhorn424** [comment]
We've discussed this. Can we close this, @gaand and @MattB-GA-Boston?

**Comment by jrhorn424** [comment]
Or do we need an action item?

**Comment by gaand** [comment]
I think we need a label, perhaps "fixed", and a column in waffle (if we use it), as the step before closed.  At least for some issues.  Perhaps we should codify whatever is decided so new instructional team members know how we handle this stuff.

**Comment by jrhorn424** [comment]
I've seen columns liked this named "Ready for Done" or something similar. You can make lots of these interstitial columns, "Ready for ..." to facilitate a [kanban pull flow](http://leankit.com/learn/kanban/kanban-pull-system/).

This may not be necessary though. I've been using issues as a discussion platform of sorts and asking for issue closure when I didn't open something.

**Comment by gaand** [comment]
Asking for closure seems fine to me.

**Comment by raq929** [comment]
I documented our current process here: https://github.com/ga-wdi-boston/instructors/wiki/ClosingIssues

**Comment by gaand** [comment]
That's good.  Let's see if we can actually make it happen 🤖 😸
